### ISSUE TITLE
translators section to translated documents

### ISSUE THREAD

**Issue by elarlang** [OWASP/ASVS#1179]
To give deserved acknowledge for translators, we should provide separate section for translators in (by current structure) `0x01-Frontispiece.md` file.

Translators should be listed only to their own language version (I don't see the need to display translators for Arabic in Spanish version of ASVS).

Additionally translators are (already) listed in general README.md file.

New section will be done for translations since v4.0.3.

Tasks:
* [ ] prepare section for translators (we can use it in `en` version files inside HTML comments in markdown file)
* [ ] section for `es`
* [ ] section for `ar`
* [ ] section for `zh-cn`

**Comment by tghosth** [comment]
I agree but can we do this from 5.0 onwards? Translators are recognised on the front page of the repo

**Comment by elarlang** [comment]
We can. Depends how much we want to get translations for v4.0.3.
